murakami.toml.example

```diff
@@ -36,7 +36,41 @@ connection-type = "wired"
 [tests]
 
   [tests.dash]
-  enabled = false
+  enabled = true
+
+  [tests.ndt5]
+  enabled = true
 
   [tests.ndt5]
   enabled = true
+
+  [tests.speedtestsingle]
+  enabled = true
+
+  [tests.speedtestmulti]
+  enabled = true
+
+  [tests.ndt5custom]
+  enabled = false
+
+  [tests.ndt7custom]
+  enabled = false
+
+  # OONI Probe checks whether your provider blocks access to sites and services. Run
+  # OONI Probe to collect evidence of internet censorship and to measure your
+  # network performance.
```

Comment:
```suggestion
  # OONI Probe to collect evidence of internet censorship. Please, be aware that:
```

Reply:
(I'm not sure whether to mention performance here and anyway dash and ndt7 are already in Murakami)